src/share/classes/com/sun/javatest/regtest/exec/Agent.java

```diff
@@ -147,7 +148,11 @@ private Agent(File dir, JDK jdk, List<String> vmOpts, Map<String, String> envVar
             // is platform-specific, and Solaris has it on by default.
             ss.setReuseAddress(false);
             InetAddress loopbackAddr = InetAddress.getLoopbackAddress();
-            ss.bind(new InetSocketAddress(loopbackAddr, /*port:*/ 0), /*backlog:*/ 1);
+            // We allow for some unexpected processes to connect to the port we listen on.
+            // We discard such connections if, after connecting, our internal handshake fails.
+            final int maxAcceptAttempts = 3;
```

Comment:
this is effectively a constant -- maybe have it as a private static final MAXACCEPTATTEMPTS = 3;

Reply:
Done. I've now moved this to a `private static final` field.